### PR TITLE
Registering TEAM codeAR

### PR DESCRIPTION
**Pull request by arnav-snowleo**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [x] Registration Pull request.

[] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

**Comment by Uyadav207** [comment]
@arnav-snowleo , Everthing Looks great. please can you fill the application checklist .

put x between [] like this [x]

**Review by Uyadav207 — APPROVED** [review]

**Comment by arnav-snowleo** [comment]
Greetings,

I have made the required changes.
Let me know if something is missing.

Regards
Arnav

**Comment by Uyadav207** [comment]
You haven't checked the list above 

wait I will edit it.

File: content/teams/codeAR/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+_**University-Name**_ : Trident Academy of Technology
+
+_**country**_ : India
+ 
+_**Technical-Skills**_ : Unity, Java, Kotlin, Firebase, Flutter, Dart
+
+_**Specialization**_ : Flutter
+
+---
+
```
Comment (Uyadav207) [inline content/teams/codeAR/index.md]: Mention member 2 Github link. same as you have mention for Leader.

File: content/teams/codeAR/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+_**Application-Type**_ :   Mobile Application 
+
+_**Idea-Description**_ :   The Scanner App will take images from the users device or camera and allow the user to crop, add scanner filters, genearte pdf file and allow sharing the file to other users using Whatsapp, email, hangouts, and other social medias. What differentiates this app from the rest of the similar apps available is that all off them are native apps and the native android apps use either external SDKs or readily available ScanLibrary for the entire functionality , but our App, will be built in Flutter and we will code the entire functionality portion ourselves.
+We will also integarte QR Scanner and Barcode Scanner which will scan Barcodes of products and detect if the product is an Indian Product or not.
+Furthermore, we plan to integrate on the fly image editing and text detection using OCR.
+
+_**Tech-Stack-Used**_ :   The Mobile App will be built in Flutter framwork with Dart Language
+
+_**GitHub-Link**_ :   " https://github.com/arnav-snowleo/ScannerApp "
```
Comment (Uyadav207) [inline content/teams/codeAR/index.md]: No quotes required .

File: content/teams/codeAR/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+_**Theme-Name**_ :  AtmaNirbhar Bharat
+
+_**Application-Type**_ :   Mobile Application 
+
+_**Idea-Description**_ :   The Scanner App will take images from the users device or camera and allow the user to crop, add scanner filters, genearte pdf file and allow sharing the file to other users using Whatsapp, email, hangouts, and other social medias. What differentiates this app from the rest of the similar apps available is that all off them are native apps and the native android apps use either external SDKs or readily available ScanLibrary for the entire functionality , but our App, will be built in Flutter and we will code the entire functionality portion ourselves.
+We will also integarte QR Scanner and Barcode Scanner which will scan Barcodes of products and detect if the product is an Indian Product or not.
+Furthermore, we plan to integrate on the fly image editing and text detection using OCR.
+
+_**Tech-Stack-Used**_ :   The Mobile App will be built in Flutter framwork with Dart Language
```
Comment (Uyadav207) [inline content/teams/codeAR/index.md]: Minor Typos in here . write Framework like this

File: content/teams/codeAR/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+github-link: "https://github.com/arnav-snowleo/ScannerApp "
+description: "Flutter App Developers"
+Team-Quote: "codeforgood"
+Team_Tech_Stack: "Flutter"
+Application_Type: "Mobile App"
+No_Of_Member: "2"
+---
+
+Hey ! We are Team codeAR. We are Tech-savvy teens who believe education without contributing to the society is meaningless and in the quest of AtmaNirbhar Bharat initiative we wish to code, contribute and make India self-reliant.
+---
```
Comment (Uyadav207) [inline content/teams/codeAR/index.md:15]: one line of spacing needed.

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by Uyadav207** [comment]
@arnav-snowleo , make these more change and then your team is ready to roll.

**Comment by arnav-snowleo** [comment]
Did the required changes.

**Review by Uyadav207 — APPROVED** [review]

**Comment by Uyadav207** [comment]
@arnav-snowleo 
 Hola  👋 ! Team  you have successfully completed the `First phase of the Hackathon`. ( We have successfully merged your PR for Registration ). 
Now we Request you to join your Team to our [**CODE CAMP GROUP 1**](https://chat.whatsapp.com/EowU6RVas3QCmtEs7q7L0H).
And you can start making your project from `21th July,2020`.
We have attached important Links. 

[**TimeLine**](https://codecamphackathon.netlify.app/TimeLine/)
[**Rule Book**](https://codecamphackathon.netlify.app/RuleBook/)
[**Prizes**](https://codecamphackathon.netlify.app/Prizes/)

See you there!
@ISTESRMNCR

**Comment by Uyadav207** [comment]
@arnav-snowleo , please join whatsapp group and say `Team name and we are in`

**Comment by arnav-snowleo** [comment]
Where do I find the Whatsapp group invite link?

**Comment by Uyadav207** [comment]
@arnav-snowleo , [**CODE CAMP GROUP 1**](https://chat.whatsapp.com/EowU6RVas3QCmtEs7q7L0H). this was the link given early. kindly be active.

**Comment by Uyadav207** [comment]
and after joining the group please reply your team name, and also say we are in NOW

**Pull request merged**